### PR TITLE
fix(examples): small bug fix.

### PR DESCRIPTION
**Pull request by haruhi55**
1. Add missing header.
1. Use average difference as the metric for unit tests.

### CODE REVIEW

File: examples/python/gemm/main.py [hunk 1]
```diff
@@ -26,7 +26,8 @@ def run_unittest(a: Tensor,
         print("\nReference:")
         print(ref_c)
 
-    if not torch.allclose(c.half(), ref_c, atol=1e-3):
+    avg_diff = (torch.sum(torch.abs(ref_c - c)) / (M * N)).item()
+    if avg_diff > 5e-2:
```
Comment (KuangjuX) [inline examples/python/gemm/main.py:30]: Can we set this as a parameter?

Comment (haruhi55) [reply examples/python/gemm/main.py]: Make sense. Done.

**Review by KuangjuX — APPROVED** [review]
LGTM
